lib/Controller/RoomController.php

```diff
@@ -469,24 +555,53 @@ protected function formatRoomV2(Room $room, ?Participant $currentParticipant): a
 			$lobbyTimer = 0;
 		}
 
+		if ($isSIPBridgeRequest) {
+			return array_merge($roomData, [
```

Comment:
Please also add information on the participant (for the case information is returned based on the PIN):
- the userid if it's a known user
- `participantType`

If you want to match web- and phone users that are invited by email, e.g. for muting web audio if dialed in by phone, something unique should be returned that is shared for both connections. You could for example use an internal user id like `invited:<email>`.

Reply:
> If you want to match web- and phone users that are invited by email,

let's focus on email invited guests for now and we figure out the logged in users once this works.

> Please also add information on the participant (for the case information is returned based on the PIN):

Opss, yeah since its a sip request we skip that, not intended on the pin verification